### DISCUSSION TITLE
setup HA (High availability) for wiki.js

### DISCUSSION THREAD

**Discussion by fandd1234** [requarks/wiki#6170] [Help / Questions] [answered]
Hi all,
Just wonder what's the best way to setup HA (High availability) for wiki.js?
I see that there's a specific part to setup HA from the config.yml file but seems like that's for PostgreSQL only.

So what if I setup the HA for the wiki.js frontend but both of them connects to my existing SQL DB.  Is it feasible?
Please share your idea/thoughts.  Thank you.

**Comment by NGPixel** [comment, marked as answer]
Only PostgreSQL is supported. Using HA with any other DB engine will result in content mismatch between instances (e.g. editing a page will only update on that instance, while other instances will keep serving the old content).
